fdbserver/include/fdbserver/IPageEncryptionKeyProvider.actor.h

```diff
@@ -384,7 +384,7 @@ class TenantAwareEncryptionKeyProvider : public IPageEncryptionKeyProvider {
 			return FDB_DEFAULT_ENCRYPT_DOMAIN_NAME;
 		}
 		if (tenantPrefixIndex.isValid()) {
-			StringRef prefix = TenantMapEntry::idToPrefix(domainId);
+			Key prefix(TenantMapEntry::idToPrefix(domainId));
```

Comment:
thought: may we want to update TenantMapEntry::idToPrefix to return `Standalone<StringRef>` instead.

Reply:
cc: @ajbeamon

Reply:
@sfc-gh-ahusain it does return `Key` which is an alias of `Standalone<StringRef>`.